luci-app-olsrd2/Makefile

```diff
@@ -0,0 +1,57 @@
+# call BuildPackage - OpenWrt buildroot signature
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luci-app-olsrd2
+PKG_VERSION:=0.2.6
+PKG_RELEASE:=14
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/luci-app-olsrd2
+  SECTION:=luci
+  CATEGORY:=LuCI
+  SUBMENU:=3. Applications
+  TITLE:=OLSR2 configuration and status module
+  MAINTAINER:=Patrick Grimm <patrick@lunatiki.de>
+  EXTRA_DEPENDS:=oonf-olsrd2, luci-mod-admin-full
```

Comment:
Extra depends? Why? This package depends on ``oonf-olsrd2``. It's not extra dependency.
I wonder, why do you need to have ``luci-mod-admin-full``?